### ISSUE TITLE
[BUG] Nature Staff unable to activate

### ISSUE THREAD

**Issue by Shibva** [DaFuqs/Spectrum#226]
**Describe the bug**
I try useing the vegetal staff but to no avail, I have vegetal in my inventory and nothing I'm doing is working. IM not sure if I'm using it wrong and I'm not to right click or its just the bug returning

**To Reproduce**
not sure how, might be due to a set of mods or one mod possibly breaking things

**Mod version**
Deeper Down Alpha18

**Comment by Shibva** [comment]
Update:

I found 2 recipies in REI that might be issues 
![image](https://github.com/DaFuqs/Spectrum/assets/38770780/8cbd9329-675d-4820-9e5b-637af41f0e43)
![image](https://github.com/DaFuqs/Spectrum/assets/38770780/2daf373f-1acb-4102-9a7a-d464cc2c567a)


not sure if these might be breaking stuff; this was in a test work where I gave myself all advancments

**Comment by DaFuqs** [comment]
I'll look into that, ty!

**Comment by Shibva** [comment]
thanks (the name would be better if it was the third time this happen as it would coincide with the Empire Strikes Back thing I tried to pull here lel)

**Comment by DaFuqs** [comment]
The recipes you listed are of blocks that do not exist in item form, therefore displaying as air. I made sure those do not show up in the future.

The staff itself works perfectly for me at least. Are you holding the right click long enough? Does ink / vegetal get consumed? What are you using it on?

**Comment by Shibva** [comment]
@DaFuqs sorry, forgot you responded to this.

ok I did some testing on the current release on curseforge since I never used it before. I now know what is going on

it appears nothing is happening when I hold right click; and by nothing I mean nothing. From using a plain instance wtih just Spectum I now know that if you try to use it, it will produce a sound when opperating regardless. In the pack I made, its not making any sort of audio altogether, suggesting that its not properly detecting that im trying to use it for some reason

my guess is that something is interfearing wtih its ability to charge and thats whats cauing the issue
